### ISSUE TITLE
Kyma installation gets stuck in local environment

### ISSUE THREAD

**Issue by valekar** [kyma-project/kyma#12021]
**Description**
OS: macOS Big Sur
minikube version: v1.22.0
kubectl Version : 1.21.4
kyma CLI version : 1.24.5

**Steps**
1. kyma provision minikube ---> this is a success
2. kyma install ---> failure, this gets stuck at Preparing installation and nothing happens even after keeping it for more than 2 hours.
3. I even tried installing using kyma install --source=local --src-path <path-to-kyma> but nothing seems to be working

Alternatives tried : I tried installing kyma-cli and kyma release 1.20.0 version. But still the result is the same. 

Any help could be really useful. I have been stuck at this step for more than 2 days.

**Comment by valekar** [comment]
Alright good news, I was able to solve this issue. The problem was with minkube. I had to add some DNS settings to minkube. Thanks to this post and @K-Visscher - https://github.com/kubernetes/minikube/issues/4589

Follow the following steps to add DNS into minkube

1. `minikube ssh`
2. `sudo vi /etc/systemd/network/10-eth1.network` add `DNS=8.8.8.8` under [Network]
3. `sudo vi /etc/systemd/network/20-dhcp.network` add `DNS=8.8.8.8` under [Network]
4. `sudo systemctl restart systemd-networkd`
5. To test it execute something that has to resolve using dns, like curl google.com or docker pull
